packages/fluentui/react-northstar/src/components/Attachment/AttachmentBody.tsx

```diff
@@ -18,10 +18,12 @@ const AttachmentBody = compose<'div', AttachmentBodyOwnProps, AttachmentBodyStyl
     className: attachmentBodyClassName,
     displayName: 'AttachmentBody',
   },
-) as ComponentWithAs<'div', AttachmentBodyProps> & { create?: ShorthandFactory<AttachmentBodyProps> };
+) as ComponentWithAs<'div', AttachmentBodyProps> & { shorthandConfig: ShorthandConfig<AttachmentBodyProps> };
 
 AttachmentBody.propTypes = commonPropTypes.createCommon();
 
-AttachmentBody.create = createShorthandFactory({ Component: AttachmentBody, mappedProp: 'content' });
+AttachmentBody.shorthandConfig = {
```

Comment:
Technically this is a breaking change, but as this was never released we are good